CodeEdit/Features/StatusBar/ViewModifiers/UpdateStatusBarInfo.swift

```diff
@@ -61,3 +65,9 @@ struct UpdateStatusBarInfo: ViewModifier {
     }
 
 }
+
+extension View {
```

Comment:
I don't know if we want this modifier applicable to every `View`

@thecoolwinter @Wouter01 Your opinion on this?

Reply:
Probably not. The new changes have reverted back to directly using the `modifier` view modifier.